deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -60,9 +60,28 @@ spec:
               - repository
               - tag
               type: object
+            sidecars:
```

Comment:
why the node side cars does not listed inside the "node" tag?

does it make sense to have "sidecars" tag for all the isde cars?